test/Solvers/linear_problems.jl

```diff
@@ -0,0 +1,298 @@
+#= 
```

Comment:
Each of these problems should be defined in a function that returns them.

Reply:
Do you think it should be something in this line
```julia
function lp1_test(factory)
    #= 
    Name: lp1
    
    Model: 
    min -4x_2 - 1
    st
    x_1 + 2x_2 <= 3
    x_1 >= 1
    x_1 >= 3
    =#

    primal_model_lp1 = TestModel{Float64}()
    
    X = MOI.add_variables(primal_model_lp1, 2)
    
    MOI.add_constraint(primal_model_lp1, 
        MOI.ScalarAffineFunction(
            MOI.ScalarAffineTerm.([1.0, 2.0], X), 0.0),
            MOI.LessThan(3.0))
    
    MOI.add_constraint(primal_model_lp1, 
        MOI.SingleVariable(X[1]),
            MOI.GreaterThan(1.0))
    
    MOI.add_constraint(primal_model_lp1, 
        MOI.SingleVariable(X[1]),
            MOI.GreaterThan(3.0))
    
    MOI.set(primal_model_lp1, 
        MOI.ObjectiveFunction{MOI.ScalarAffineFunction{Float64}}(), 
        MOI.ScalarAffineFunction(MOI.ScalarAffineTerm.([-4.0], [X[2]]), -1.0)
        )
    
    MOI.set(primal_model_lp1, MOI.ObjectiveSense(), MOI.MIN_SENSE)
    
    dual_model_lp1 = dualize(primal_model_lp1)

    @test test_strong_duality(primal_model_lp1, dual_model_lp1.dual_model, factory)
    
    # Test if the dual model is the expected dual model

end
```
I the last part I should test if the dual model is what I expected